submissions/motunrayoIlawole/challenge1-arrays/question3.js

```diff
@@ -0,0 +1,25 @@
+function solution (nums) {
+    // get all the positive integers sorted
+    let sortedNums = nums.filter(num => num >= 1).sort((a, b) => a - b);
```

Comment:
Also very nice and simple. 
Your time complexity however is O(nlogn) as that's how long the sorting algorithm would take.
The space complexity is O(n), since you are creating another array with a count less than or equal to your given array. i.e. `sortedNums`.